pkg/migrations/00011_payload-expiration.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE gateway_envelopes
+    ADD COLUMN expiry BIGINT;
```

Comment:
Looking at the generated go code, it seems this will be set to nil to old rows? Just to take it in account.

Reply:
correct. 0 meens infinite, NULL means it hasn't been set. It requires sophisticated non-sql code to inspect the payload and determine whether it is a commit or not.